awscli/customizations/eks/get_token.py

```diff
@@ -29,6 +29,8 @@
 # Presigned url timeout in seconds
 URL_TIMEOUT = 60
 
+TOKEN_EXPIRATION_MINS = 14
```

Comment:
Can you elaborate on how `14` was chosen here? I looked at this PR: https://github.com/kubernetes-sigs/aws-iam-authenticator/pull/160 and while it was originally hardcoded `14` as well, it was updated to subtract `1` minute off of the original presigned URL expiration time. I wonder if we should follow suit here?

Reply:
Unlike pre-signed URLs in S3, STS does not honor X-Amz-Expires header (which is part of the sigv4 auth headers) and they have a default timeout of 15 minutes.

Reply:
Gotcha. Thanks for clarifying.